analyzers/src/SonarAnalyzer.CFG/LiveVariableAnalysis/RoslynLiveVariableAnalysis.cs

```diff
@@ -152,6 +152,34 @@ private static IEnumerable<ControlFlowRegion> CatchOrFilterRegions(ControlFlowRe
             }
         }
 
+        private static ISymbol OriginalDeclaration(IOperation originalOperation)
+        {
+            ISymbol symbol;
+            if (originalOperation.IsAnyKind(OperationKindEx.MethodBody, OperationKindEx.Block, OperationKindEx.ConstructorBody))
+            {
+                var operationWrapper = new IOperationWrapperSonar(originalOperation);
+                var syntax = originalOperation.Syntax;
+                if (syntax.IsKind(SyntaxKindEx.ArrowExpressionClause))
+                {
+                    syntax = syntax.Parent;
+                }
+
+                symbol = operationWrapper.SemanticModel.GetDeclaredSymbol(syntax);
+            }
+            else
+            {
+                symbol = originalOperation switch
+                {
+                    var _ when ILocalFunctionOperationWrapper.IsInstance(originalOperation) => ILocalFunctionOperationWrapper.FromOperation(originalOperation).Symbol,
+                    var _ when IFlowAnonymousFunctionOperationWrapper.IsInstance(originalOperation) => IFlowAnonymousFunctionOperationWrapper.FromOperation(originalOperation).Symbol,
+                    var _ when IAnonymousFunctionOperationWrapper.IsInstance(originalOperation) => IAnonymousFunctionOperationWrapper.FromOperation(originalOperation).Symbol,
+                    _ => null
```

Comment:
This `null` will cause NRE way later, harder to diagnost. We should throw instead.